openapi/api.yml

```diff
@@ -146,7 +146,8 @@ paths:
             application/json:
               schema:
                 type: array
-                $ref: '#/components/schemas/account'
+                items:
```

Comment:
Please never say "I know you know what you're doing" because then you'll stop reviewing my stuff closely. And I do still make mistakes plenty often enough ;)

What we originally had was this:
```
              schema:
                type: array
                $ref: '#/components/schemas/account'
```

If you look at the schema definition at the bottom, we have this:

```
components:
  schemas:
    ...
    account:
      type: object
      properties:
        ...
```

If we decompose the referenced schema, we effectively end up with:

```
schema:
  type: array
  type: object
  ...
```

Which doesn't make sense.

The `/accounts/fund/{fund_short_name}` endpoint returns a list of accounts, so by moving the referenced schema into the `items` block, we essentially end up with:

```
              schema:
                type: array
                items:
                  type: object
                  ...
```

Which is what we actually want.

---

P.S. I only noticed this because my IDE (PyCharm) automatically applies a OpenAPI 3.0 JSON Schema when I am editing the `api.yml`, which shows me this - because `$ref` and `type` are mutually-exclusive.

![image](https://github.com/user-attachments/assets/756ec5d8-707b-4f77-b94c-5f86e26a8967)